src/test/java/net/library/controller/UserControllerTest.java

```diff
@@ -31,34 +35,226 @@ class UserControllerTest {
     @Autowired
     private UserService service;
 
-    @Autowired
-    public UserControllerTest(UserService service) {
-        this.service = service;
+    @BeforeEach
+    void clean(){
+        service.deleteAll();
     }
 
     @Test
-    void getUsers() throws Exception {
-        var user1 = new User("Smith1", "John1", 12);
-        var user2 = new User("Smith2", "John2", 22);
-        var user3 = new User("Smith3", "John3", 32);
+    void getAllUsers() throws Exception {
+        final var username = "Alelxo";
+        final var surname = "Bur";
+        final var name = "Alex";
+        final var email = "efaf";
+        final var phoneNumber = "380679920267";
+        final var address = "assfasfd";
+        final var xCorrelationId = getUUID();
 
-        var users = new ArrayList<>();
-        users.add(user1);
-        users.add(user2);
-        users.add(user3);
+        mvc.perform(MockMvcRequestBuilders.post(GLOBAL_BASE_URI + USERS)
+                        .header(CORRELATION_ID_HEADER_NAME, xCorrelationId)
+                        .contentType("application/json")
+                        .content("{ \"username\": \"Alelxo\", \"surname\": \"Bur\", " +
+                                "\"name\": \"Alex\", \"email\": \"efaf\", \"phoneNumber\":\"380679920267\", " +
+                                "\"address\":\"assfasfd\"}"))
+                .andExpect(status().isOk());
 
-        mvc.perform(MockMvcRequestBuilders.get(BASE_URI + USERS))
+        mvc.perform(MockMvcRequestBuilders.get(GLOBAL_BASE_URI + USERS))
                 .andExpect(status().isOk())
-                .andExpect(content().string(objectMapper.writeValueAsString(users)));
+                .andExpect(jsonPath("$.data[0].username", is(username)))
+                .andExpect(jsonPath("$.data[0].surname", is(surname)))
+                .andExpect(jsonPath("$.data[0].name", is(name)))
+                .andExpect(jsonPath("$.data[0].email", is(email)))
+                .andExpect(jsonPath("$.data[0].phoneNumber", is(phoneNumber)))
+                .andExpect(jsonPath("$.data[0].address", is(address)));
+    }
+
+    @Test
+    void excessiveLength13MsisdnValidator() throws Exception {
+        final var xCorrelationId = getUUID();
+
+        mvc.perform(MockMvcRequestBuilders.post(GLOBAL_BASE_URI + USERS)
+                        .header(CORRELATION_ID_HEADER_NAME, xCorrelationId)
+                        .contentType("application/json")
+                        .content("{ \"username\": \"Book Title\", \"surname\": \"Book Author\", " +
```

Comment:
Don't do it. Use POJO to create a structure and move forward with converting.
Also rename the test name